MapboxNavigation/NavigationMapView.swift

```diff
@@ -719,6 +722,55 @@ open class NavigationMapView: MGLMapView {
         }
     }
     
+    public func showVoiceInstructionsOnMap(routeProgress: RouteProgress) {
+        guard let style = style else {
+            return
+        }
+        
+        let instructions = Array(routeProgress.route.legs.flatMap {
+            return $0.steps.flatMap { (step: RouteStep) in
+                return step.instructionsSpokenAlongStep.flatMap {
+                    return $0.flatMap { (instruction: SpokenInstruction) in
+                        return (Polyline(step.coordinates!).coordinateFromStart(distance: instruction.distanceAlongStep)!, instruction.text)
```

Comment:
This is off by one, since the spoken instructions pertain to the following step.